demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/sample/ModalExperiment.kt

```diff
@@ -0,0 +1,110 @@
+package com.bumble.appyx.interactions.sample
+
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.compose.foundation.gestures.detectDragGestures
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.unit.dp
+import com.bumble.appyx.components.modal.Modal
+import com.bumble.appyx.components.modal.ModalModel
+import com.bumble.appyx.components.modal.ui.ModalMotionController
+import com.bumble.appyx.interactions.AppyxLogger
+import com.bumble.appyx.interactions.core.ui.context.UiContext
+import com.bumble.appyx.interactions.core.ui.gesture.GestureSettleConfig
+import com.bumble.appyx.interactions.core.ui.helper.InteractionModelSetup
+import com.bumble.appyx.interactions.sample.android.Children
+import com.bumble.appyx.interactions.sample.android.Element
+import com.bumble.appyx.transitionmodel.BaseMotionController
+import com.bumble.appyx.interactions.sample.InteractionTarget as Target
+
+@ExperimentalMaterialApi
+@Composable
+@Suppress("LongMethod", "MagicNumber")
+fun ModalExperiment(
+    modifier: Modifier = Modifier,
+    motionController: (UiContext) -> BaseMotionController<Target, ModalModel.State<Target>, *, *>
+) {
+    val items = listOf(
```

Comment:
I think we don't need this many elements for a modal :D

Reply:
I just loved the process of discarding them 😅